server/monitor_test.go

```diff
@@ -1313,18 +1313,18 @@ func TestConcurrentMonitoring(t *testing.T) {
 			for i := 0; i < 150; i++ {
 				resp, err := http.Get(url + endpoint)
 				if err != nil {
-					t.Fatalf("Expected no error: Got %v\n", err)
```

Comment:
@wallyqs I was going to fix that. This go routine needs to send error to an error channel that will be checked at the end of the test. It seems that staticcheck starts to report that and it did not. This is a real issue in that t.Fatalf must not be called in a go routine otherwise it will produce a race.